docs/quickstarts/packet.md

```diff
@@ -5,134 +5,117 @@
-  key](https://www.packet.com/developers/docs/API/getting-started/)).
-* AWS account and IAM credentials (optional for Route53 DNS configuration).
-* AWS Route53 DNS Zone (registered Domain Name or delegated subdomain).
-* Terraform v0.12.x and [terraform-provider-ct](https://github.com/poseidon/terraform-provider-ct) v0.5.0
-  installed locally.
-* Local BGP enabled. More information on how to enable Local BGP for the Packet Project is found in
-  the [Packet support document](https://support.packet.com/kb/articles/bgp).
-* An SSH key pair for management access.
-* `kubectl` installed locally to access the Kubernetes cluster.
+* A Packet account with
+  [local BGP](https://www.packet.com/developers/docs/network/advanced/local-and-global-bgp/)
+  enabled.
+* A Packet project ID.
+* A Packet API key with access to the relevant project.
+* An AWS account.
+* An AWS
+  [access key ID and secret](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html).
+* An AWS Route 53 zone (can be a subdomain).
+* An SSH key pair for accessing the cluster nodes.
+* Terraform `v0.12.x` installed.
+* [terraform-provider-ct](https://github.com/poseidon/terraform-provider-ct) `v0.5.0` installed.
+* `kubectl` installed.
 
 ## Steps
 
 ### Step 1: Install lokoctl
 
-lokoctl is a command-line interface for Lokomotive.
-
-To install `lokoctl`, follow the instructions in the [lokoctl installation](../installer/lokoctl.md)
-guide.
-
-### Step 2: Set up a working directory
+`lokoctl` is the command-line interface for managing Lokomotive clusters.
 
-It's better to start fresh in a new working directory, as the state of the cluster is stored in this
-directory.
-
-This also makes the cleanup task easier.
+Download the `lokoctl` binary for your platform:
 
 ```console
-mkdir -p lokomotive-infra/mycluster
-cd lokomotive-infra/mycluster
-```
+# Linux
+wget https://github.com/kinvolk/lokomotive/releases/download/v0.1.0/lokoctl_0.1.0_linux_amd64.tar.gz
 
-### Step 3: Set up credentials from environment variables
-
-#### Packet
-
-* Log in to your Packet account and obtain the Project ID from the `Project Settings` tab.
-* Obtain an API key from the `User Settings` menu.
-* Set the environment variable `PACKET_AUTH_TOKEN` with the API key.
-
-```console
-export PACKET_AUTH_TOKEN=<PACKET_API_KEY>
+# macOS
+wget https://github.com/kinvolk/lokomotive/releases/download/v0.1.0/lokoctl_0.1.0_darwin_amd64.tar.gz
 ```
-#### AWS
 
-Lokomotive requires AWS credentials for configuring Route53 DNS. To manually configure DNS entries
-refer the DNS configuration settings for Packet(Add link).
+Extract the binary and copy it to a place under your `$PATH`:
 
 ```console
-export AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
-export AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+tar zxvf lokoctl_0.1.0_linux_amd64.tar.gz
+sudo cp lokoctl_0.1.0_linux_amd64/lokoctl /usr/local/bin
```

Comment:
Maybe add a line to cleanup the tar.gz and folder created?

Reply:
Hm, we could provide a docker image to run lokoctl, which would include Terraform binary and all provider binaries.

Reply:
Definitely out of scope for this PR, IMHO.

Feel free to create an issue about it, if you want, and we can discuss there (spoiler alert: I strongly disagree). But if you want to talk about **that**, I'd ask to shift the conversation to an issue and not this PR, IMHO, to keep comment in-scope for this PR.

Reply:
>Maybe add a line to cleanup the tar.gz and folder created?

Done.

>Hm, we could provide a docker image to run lokoctl, which would include Terraform binary and all provider binaries.

I wouldn't do that. This adds another very tricky layer to the tutorial (Docker) with lots of things which could go wrong. I'm all for shipping the right providers with `lokoctl`, but I think we should embed them in the binary if we do.